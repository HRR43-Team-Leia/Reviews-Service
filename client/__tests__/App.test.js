// import React from 'react';
// import Enzyme from 'enzyme';
// import { shallow } from 'enzyme'
// import Adapter from 'enzyme-adapter-react-16.1';
// import App from '../components/App.jsx';

// Enzyme.configure({ adapter: new Adapter() })

// describe("App", () => {
//   const expId = 5
//   const wrapper = shallow(<App expId={expId} />)

//   it('App should render', () => {
//     expect(wrapper).toBeDefined();
//   });

//   it('App render should match snapshot', () => {
//     expect(wrapper).toMatchSnapshot()
//   });
// })
