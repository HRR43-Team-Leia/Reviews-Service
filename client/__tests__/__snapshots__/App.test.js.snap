// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App render should match snapshot 1`] = `
<div>
  <div
    className="reviewsModule"
  >
    <GuestReviews
      overallStar={0}
    />
    <div
      className="rightSide"
    >
      <div>
        <ReviewList
          reviews={Array []}
        />
      </div>
      <div
        className="pageSelector"
      >
        <PageSelector
          currPage={1}
          jumpToPage={[Function]}
          totalPages={0}
        />
      </div>
    </div>
  </div>
</div>
`;
